src/content/reference/kubectl-gs/get-clusters.md

```diff
@@ -70,7 +70,7 @@ Here we document the flags that have a particular meaning for the `get clusters`
 
 To inspect a cluster's main custom resource in YAML notation, add the `--output yaml` flag (or `-o yaml` in short) to the command.
 
-The following example command would print the main resource for cluster `ab12c`. On AWS that would be the [AWSCluster](/reference/management-api/awsclusters.infrastructure.giantswarm.io/) resource printed. On Azure, it would return the [Cluster](/reference/management-api/clusters.cluster.x-k8s.io/) resource.
+The following example command would print the main resource for cluster `ab12c`. It would return the [Cluster](/reference/management-api/clusters.cluster.x-k8s.io/) resource.
```

Comment:
This way, if the linked page goes away or is renamed. Building the docs will fail until someone fixed the link.

```suggestion
The following example command would print the main resource for cluster `ab12c`. It would return the [Cluster]({{< relref "/content/reference/management-api/clusters.cluster.x-k8s.io.md" >}}) resource.
```

Reply:
Awesome! Thank you